rosette/h/Heap.h

```diff
@@ -96,9 +86,8 @@ class Heap {
     Ob* copyAndForward(Ob*);
     void traverseRootSets(RootSet_Fn);
 
-    friend Ob* Ob::relocate(EMPTY);
+    //friend Ob* Ob::relocate(EMPTY);
```

Comment:
Please just delete the line.

Reply:
Thanks, I forgot to do that. Good catch.